addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1843,6 +1843,11 @@ export function splitElement(element, offset) {
         index < offset ? before.appendChild(child) : after.appendChild(child);
         index++;
     }
+    const blockEl = closestBlock(after);
+    if (isShrunkBlock(blockEl)) {
+        const br = document.createElement('br');
+        after.appendChild(br);
+    }
```

Comment:
Give some scenario why are we adding this, in form of docString.

Reply:
@shsa-odoo Done!